kiwi/container/oci.py

```diff
@@ -207,6 +214,23 @@ def pack_image_to_file(self, filename):
             image_dir, xz_options=self.xz_options
         )
 
+    def _get_source(self):
+        if Defaults.is_buildservice_worker():
+            with open(os.sep + Defaults.get_buildservice_env_name()) as env:
+                for line in env:
+                    if line.startswith('BUILD_DISTURL') and '=' in line:
+                        return line.split('=')[1].strip()
+                log.warning('Could not find DISTURL inside .buildenv')
+
+        return None
+
+    def _get_source_label(self):
+        source = self._get_source()
+        if source:
+            return ''.join([
+                '--config.label=org.opencontainers.image.source=', source
+            ])
+
```

Comment:
not sure, but maybe the two private methods _get_source and _get_source_label could be combined into one _get_buildservice_source_label ? imho _get_source only exists to build the label right ? so I did not understand why an extra method is needed. I might be wrong on that though

Reply:
I had splitted them because in my initial attempt the _get_source method was implemented inside kiwi/defaults.py, but later I thought that there was no need for a generic access to that value since I don't think we will need it for anything else. You are right they can be combined.